hq/app/aws/AWS.scala

```diff
@@ -20,4 +28,61 @@ object AWS {
       Failure.awsAccountNotFound(accountId).attempt
     )
   }
+
+  def inspectorClients(configuration: Configuration): Map[(String, Regions), AmazonInspectorAsync] = Config.getAwsAccounts(configuration).flatMap(account => Regions.values().map(
+    region => {
+      val auth = AWS.credentialsProvider(account)
+      val inspectorClient = AmazonInspectorAsyncClientBuilder.standard()
+        .withCredentials(auth)
+        .withRegion(region)
+        .build()
+      (account.id, region) -> inspectorClient
+    }
+  )).toMap
+
+
+  def ec2Clients(configuration: Configuration): Map[(String, Regions), AmazonEC2Async] = Config.getAwsAccounts(configuration).flatMap(account => Regions.values().map(
```

Comment:
I'm not crazy about the duplication here (on top of the codestyle, as mentioned above).

Some options I have considered follow. I think 3 is my favourite for the ones with all regions. For the ones that only take a single region there's no need to do the inner map. A simpler 1 or 2 would be best for them.

```scala

  def inspectorClients(configuration: Configuration): Map[(String, Regions), AmazonInspectorAsync] = {
    Config.getAwsAccounts(configuration).flatMap { account =>
      Regions.values().map { region =>
        val inspectorClient = AmazonInspectorAsyncClientBuilder.standard()
          .withCredentials(AWS.credentialsProvider(account))
          .withRegion(region)
          .build()
        (account.id, region) -> inspectorClient
      }
    }.toMap
  }

  def inspectorClients2(configuration: Configuration): Map[(String, Regions), AmazonInspectorAsync] = {
    (for {
      account <- Config.getAwsAccounts(configuration)
      region <- Regions.values()
    } yield {
      val inspectorClient = AmazonInspectorAsyncClientBuilder.standard()
        .withCredentials(AWS.credentialsProvider(account))
        .withRegion(region)
        .build()
      (account.id, region) -> inspectorClient
    }).toMap
  }

  def inspectorClients3(configuration: Configuration): Map[(String, Regions), AmazonInspectorAsync] = {
    clientsForAllRegions(configuration) { (account, region) =>
      AmazonInspectorAsyncClientBuilder.standard()
        .withCredentials(AWS.credentialsProvider(account))
        .withRegion(region)
        .build()
    }
  }

  def clientsForAllRegions[A](configuration: Configuration)(makeClient: (AwsAccount, Regions) => A): Map[(String, Regions), A] = {
    val clients = for {
      account <- Config.getAwsAccounts(configuration)
      region <- Regions.values()
    } yield {
      (account.id -> region) -> makeClient(account, region)
    }
    clients.toMap
  }

```